qml/CoverPage.qml

```diff
@@ -45,22 +45,29 @@ CoverBackground {
     ListView {
         id: fileListView
 
-        property real itemHeight: Theme.iconSizeSmall + Theme.paddingSmall*2
-
+        property int itemHeight: height/maxItemCount
+        property int maxItemCount: Math.round(height/(dummy.height + Theme.paddingSmall))
         clip: true
         interactive: false
         model: window.fileListModel
         visible: window.documentItem === null
-        y: Theme.paddingLarge
-        width: parent.width
-        height: 7*itemHeight
+        anchors {
+            fill: parent
+            topMargin: Theme.paddingLarge
+            bottomMargin: Theme.paddingLarge
+        }
 
         delegate: CoverFileItem {
             width: fileListView.width
             height: fileListView.itemHeight
             text: model.fileName
             iconSource: window.mimeToIcon(model.fileMimeType)
         }
+        Label {
```

Comment:
I don't know if it's a good suggestion, but anyway:

Would it be better to create here a dummy CoverFileItem instance, so like that if later on, someone who makes changes in the layout of CoverFileItem (and its height is not a label height anymore), won't have to update here also ?

This suggestion, though requires to add ```height: label.height``` in CoverFileItem.qml.